ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -1,251 +1,317 @@
+                onMenuButtonClicked: {
+                    statusChatListCategoryItem.setupPopup()
+                    highlighted = true
+                    menuButton.highlighted = true
+                    let p = menuButton.mapToItem(statusChatListCategoryItem, menuButton.x, menuButton.y)
+                    let menuWidth = categoryPopupMenuSlot.item.width
+                    categoryPopupMenuSlot.item.popup()
                 }
-                onActiveChanged: d.destinationPosition = -1
-
-                StatusChatListItem {
-                    id: statusChatListItem
-
-                    width: parent.width
-                    opacity: dragSensor.active ? 0.0 : 1.0
-                    originalOrder: model.position
-                    chatId: model.itemId
-                    categoryId: model.parentItemId? model.parentItemId : ""
-                    name: model.name
-                    type: !!model.type ? model.type : StatusChatListItem.Type.CommunityChat
-                    muted: model.muted
-                    hasUnreadMessages: model.hasUnreadMessages
-                    notificationsCount: model.notificationsCount
-                    highlightWhenCreated: !!model.highlight
-                    selected: (model.active && root.highlightItem)
-                    asset.emoji: !!model.emoji ? model.emoji : ""
-                    asset.color: !!model.color ? model.color : Theme.palette.userCustomizationColors[model.colorId]
-                    asset.isImage: model.icon.includes("data")
-                    asset.name: model.icon
-                    ringSettings.ringSpecModel: type === StatusChatListItem.Type.OneToOneChat && root.isEnsVerified(chatId) ? undefined : model.colorHash
-                    onlineStatus: !!model.onlineStatus ? model.onlineStatus : StatusChatListItem.OnlineStatus.Inactive
-
-                    sensor.cursorShape: dragSensor.cursorShape
-
-                    Connections {
-                        target: statusChatListItem
-                        enabled: root.draggableItems
-
-                        function onOriginalOrderChanged() {
-                            Qt.callLater(() => {
-                                if (!delegateModel)
-                                    return
-
-                                delegateModel.items.setGroups(0, delegateModel.items.count, "unsorted")
-                            })
-                        }
+                onAddButtonClicked: {
+                    root.categoryAddButtonClicked(categoryId)
+                }
+            }
+        }
 
+        delegate: Loader {
+            id: chatLoader
+            active: model.type !== LocalUtils.Constants.chatType.category && model.categoryOpened
+            // TODO fix height not adjusting
+            // This below doesn't work well, the height stays at 0 after reopening
+            // height: active && item ? item.height : 0
```

Comment:
Is this still broken?

Reply:
Yes. If you have suggestions or tips I would greatly appreciate. It's annoying, when I collapse the item, it disappears, but the height doesn't.
If I put that line, then the chat items do disappear, but when reopening, their height stay at 0

Reply:
`visible: active`?

Reply:
Alright, will try to make these UI fixes afterwards